.gitlab-ci.yml

```diff
@@ -32,6 +33,18 @@ ci-image:
     - docker build --tag $CI_IMAGE_DOCKER -f Dockerfile.gitlab .
     - docker push $CI_IMAGE_DOCKER
 
+# SECURITY
+
+create_key:
+  when: manual
+  tags: [ "runner:docker", "size:large" ]
+  variables:
+    PROJECT_NAME: "dd-sdk-android"
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/agent-key-management-tools/create-gpg-key:1
```

Comment:
is this image generated/provided by AWS KMS or by us?

Reply:
This is an image provided by us in a different repo (cf project `agent-key-management-tools`)